File: backend/src/main/kotlin/com/corina/auto_manager/models/Vehicle.kt
package com.corina.auto_manager.models

class Vehicle(
    var regNo: String,
    var brand: String,
    var km: Int,
    var itp: VehicleAttribute?,
    var rca: VehicleAttribute?,
    var vignette: VehicleAttribute?,
    var oilChange: VehicleAttribute?
) {



}

File: frontend/src/types/vehicle.ts
import type { VehicleAttribute } from "./vehicleAttribute";

export interface Vehicle {
  regNo: string;
  brand: string;
  km: number;
  itp?: VehicleAttribute;
  rca?: VehicleAttribute;
  vignette?: VehicleAttribute;
  oilChange?: VehicleAttribute;
}
